slave/components/quality_controller/error_correction.go

```diff
@@ -55,7 +55,7 @@ func (e *errorCorrection) Process() (string, error) {
 			"-d", path.Join(constants.BaseOut, "canu-corr"),
 			"-p", "run1",
 			fmt.Sprintf("genomeSize=%s", e.config.Assemblers.Flye.GenomeSize),
-			"-nanopore-raw", e.toDecontaminate,
+			"-nanopore-raw", path.Join(constants.BaseOut, "trimmed.fastq"),
```

Comment:
I don't think the error correction step should know the filename like this. It is given the filename through:
```go
// toDecontaminate represents the path to the file to decontaminate
toDecontaminate string
```